src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/impl/updaters/transformers/ProjectDetailsToModuleDetailsTransformer.kt

```diff
@@ -35,6 +35,7 @@ internal class ProjectDetailsToModuleDetailsTransformer(
       moduleDependencies = allDependencies.targetDependencies.map { it.uri },
       defaultJdkName = projectDetails.defaultJdkName,
       jvmBinaryJars = jvmBinaryJarsIndex[targetId].orEmpty(),
+      goLibraries = projectDetails.libraries.orEmpty()
```

Comment:
Tu jest ta linijka, o której mówiłem, że dla każdego targetu / modułu GO dodajemy wszystkie biblioteki dostępne w projekcie, pewnie chcemy filtrować, jako że w target.dependencies mamy list target id od których zależymy (wewnętrzne i zewnętrzne zależności), to możemy właśnie zrobić coś w stylu:

```suggestion
      goLibraries = projectDetails.libraries?.filter { 
        it.id in target.dependencies
      }.orEmpty()
```

Reply:
Drugą opcją (wg mnie ładniejszą) jest całkowicie usunąć `goLibraries` z `ModuleDetails` jako, że trochę się pokrywamy z libraryDependencies (tam zapiszemy tylko targetId dla naszych bibliotek, ale to nie problem) - wyciąganie potem odpowiednich danych możemy zrobić w ModuleDetailsToGoModuleTransformer